environment.yaml

```diff
@@ -31,10 +31,8 @@ dependencies:
 - wget=1.20.1
 - pip:
   - amazon-kclpy==2.0.1
-  - crontab==0.22.6
-  - domain-utils==0.7.1
+  - crontab==0.22.8
   - flask-cors==3.0.8
-  - moto-ext==1.3.15.15
-  - plyvel==1.2.0
```

Comment:
Assuming you didn't manually edit this yourself, something has gone wrong with the repin script. We've lost domain-utils and plyvel.

Reply:
Hmm maybe it's because my conda env wasn't active?

Reply:
I've updated conda and retried it with the openwpm env active. No change. Seems like the script may have a bug. Note that both of these are python dependencies, so it could be related to that.

Reply:
activating openwpm env won't change anything. the script activates it for itself (https://github.com/mozilla/OpenWPM/blob/master/scripts/repin.sh#L10)

first debug step. run 
```
# Create openwpm env with unpinned yaml file
conda env create --force -q -f environment-unpinned.yaml

# Activate
conda activate openwpm
```

Then `conda list` and see if domain-utils and plyvel are in the list.

Then, next steps from repin

```
# Adding dev dependencies to environment
conda env update -f environment-unpinned-dev.yaml

# Export the environment including manually specify channels
conda env export --no-builds --override-channels -c conda-forge -c main -f ../environment.yaml
```

Then have a look at `environment.yaml`. If plyvel and domain-utils are in there (which I expect they will be) then the problem is with `prune-environment.py`.

I'm expecting them to be successfully in because it's much more likely that the custom script I wrote to clean up the environment is broken rather than conda being broken.

Can you post the output from `../environment.yaml` before pruning.

Reply:
To avoid this getting lost in a comment thread I followed up directly in the PR: https://github.com/mozilla/OpenWPM/pull/682#issuecomment-645648939